core/examples/luigi-sample-angular/src/luigi-config/extended/projectDetailNav.js

```diff
@@ -64,6 +64,12 @@ export const projectDetailNavStructure = projectId => [
     viewUrl: '/sampleapp.html#/projects/' + projectId + '/developers',
     icon: '/assets/favicon-sap.ico'
   },
+  {
+    pathSegment: 'clientPermissionsTets',
+    label: 'ClientPermissionsTets',
+    viewUrl: '/sampleapp.html#/projects/pr1',
+    hideFromNav: true
```

Comment:
Do you think we should expose the node so that everyone can see this feature in action?

Reply:
We have it in action in project 1 in currentLocale. It's just true. But we can discuss it in scrum.

Reply:
True. ok for me to leave it as it is.